pkg/serverstate/statetest/test_add_on.go

```diff
@@ -218,17 +218,31 @@ My favorite add-on README.
 		require.Equal(testAddOn.TerraformNocodeModule.Source, actualAddOn.TerraformNocodeModule.Source)
 		require.Equal(testAddOn.TerraformNocodeModule.Version, actualAddOn.TerraformNocodeModule.Version)
 
+		updatedAddOnName := "your updated friendly neighborhood add-on"
+		updatedAddOn, err := s.AddOnUpdate(ctx,
+			&pb.AddOn{
+				Name: updatedAddOnName,
+			},
+			&pb.Ref_AddOn{
+				Identifier: &pb.Ref_AddOn_Name{
+					Name: testAddOn.Name,
+				},
+			},
+		)
+		require.NoError(err)
+		require.NotNil(updatedAddOn)
```

Comment:
Can we do another get after the update, to check that the updated properties are being returned?

Reply:
Added!